packages/@react-aria/dnd/docs/useDroppableCollection.mdx

```diff
@@ -0,0 +1,1080 @@
+
+```tsx example
+function Example() {
+  ///- begin collapse -///
+  let list = useListData({});
+  let onRootDrop = async e => {
+    let items = await Promise.all(
+      e.items
+        .map(async item => ({
+          id: Math.random(),
+          url: URL.createObjectURL(await item.getFile()),
+          name: item.name
+        }))
+    );
+
+    list.prepend(...items);
+  };
+  ///- end collapse -///
+  // ...
+
+  return (
+    <ListBox
+      aria-label="Images"
+      items={list.items}
+      /*- begin highlight -*/
+      getDropOperation={() => 'copy'}
+      /*- end highlight -*/
+      acceptedDragTypes={['image/png']}
+      onRootDrop={onRootDrop}>
+      {item => (
+        <Item>
+          <div className="image-item">
+            <img src={item.url} />
+            <span>{item.name}</span>
+          </div>
+        </Item>
+      )}
+    </ListBox>
+  );
+}
+```
+
+### Drop events
+
+Drop events such as `onInsert`, `onItemDrop`, etc. also include the `dropOperation`. This can be used to perform different actions accordingly, for example, when communicating with a backend API.
+
+```tsx
+let onItemDrop = async (e) => {
+  let data = JSON.parse(await e.items[0].getText('my-app-file'));
+  /*- begin highlight -*/
+  switch (e.dropOperation) {
+    case 'move':
+      MyAppFileService.move(data.filePath, props.filePath);
+      break;
+    case 'copy':
+      MyAppFileService.copy(data.filePath, props.filePath);
+      break;
+    case 'link':
+      MyAppFileService.link(data.filePath, props.filePath);
+      break;
+  }
+  /*- end highlight -*/
+};
+```
+
+## Low level API
+
+The above examples have used high level events such as `onInsert`, `onItemDrop`, and `onReorder`, along with props such as `acceptedDragTypes`. Based on these props, `useDroppableCollection` automatically determines whether drag data is accepted and where (e.g. on items, between items, etc.). It also automatically filters the dropped items based on their types. For more complex scenarios, the lower level `getDropOperation` and `onDrop` functions can be used instead.
+
+To programmatically determine whether a drop is accepted based on the dragged types, target key, and drop position, implement the `getDropOperation` function. `acceptedDragTypes` and `shouldAcceptItemDrop` may also be used in combination with `getDropOperation` as a pre-filter. See the section on [drop operations](#drop-operations) above for more details.
+
+To handle all accepted drops in a single function, implement the `onDrop` event rather than `onInsert`, `onItemDrop`, etc. When defined, this overrides any other drop handlers. The provided <TypeLink links={sharedDocs.links} type={sharedDocs.links[sharedDocs.exports.DroppableCollectionDropEvent.id]} /> includes details on the drop target, dropped items, drop operation, etc.
```

Comment:
Maybe we should include a warning symbol or bold, something to really draw attention to 'this overrides and other drop handlers'
right now, it's easily lost in the paragraph but it's quite important

Reply:
I tried it but it just looked weird. I think people will figure it out, but we can see.